ui/org.eclipse.pde.bnd.ui/README.MD

```diff
@@ -46,6 +46,8 @@ public class BndPluginAdapter implements IAdapterFactory {
 }
 ```
 
+Also see `BndProjectManager.java~.
```

Comment:
Can you tell what exactly this reference is meant for? It should also probably be a link to the source in the repo.

Reply:
As mentioned earlier. I as a noob wished the readme points me to existing files which are relevant. 
Sure I can link. But maybe let's talk about about your desired readme style.